loopy/statistics.py

```diff
@@ -916,14 +931,17 @@ def __init__(self, knl, callables_table, kernel_rec,
     def combine(self, values):
         return sum(values)
 
-    def map_constant(self, expr):
+    def map_constant(self, expr, tags=None):
```

Comment:
Why the pervasive `tags=None`? The default risks losing the tag as you traverse. IMO, there should not be a default.

Reply:
I changed it in [df7ed9b](https://github.com/inducer/loopy/pull/688/commits/df7ed9b5dee8fadf1c984a22985edfce7ecbf454) to not use a default argument anymore.